File: Unomart/src/app/models/order.ts
export class OrderItem{
    IID:string = '';
    OID:string = '';
    ItemAmount:number = 0;
    ItemCost?:number = 0;
    ItemImg?:string = '';
    ItemName:string = '';
}

export class Order{
    oid:string = '';
    uid:string = '';
    discount:number = 0;
    deliveryCharge:number = 0;
    deliveryAddress:string = '';
    total:number = 0;
    items:number = 0;
    dateTime:Date = new Date();
}

File: Backend/Models/Order.cs
namespace Unomart.Models
{
    public class OrderItem
    {
        public string IID { get; set; }
        public string OID { get; set; }
        public int ItemAmount { get; set; }
        public string ItemName { get; set; }
        public string ItemImage { get; set; }
        public float ItemCost { get; set; }
    }
    public class Order
    {
        public string OID { get; set; }
        public string UID { get; set; }
        public string DeliveryAddress { get; set; }
        public float DeliveryCharge { get; set; }
        public float Discount { get; set; }
        public float Total { get; set; }
        public int Items { get; set; }
        public DateTime DateTime { get; set; }
    }
}
